src/arrays/Arrays.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace arrays;
+
+class Arrays implements ArraysInterface
+{
+    /**
+     * The $input variable contains an array of digits
+     * Return an array which will contain the same digits but repetitive by its value
+     * without changing the order.
+     * Example: [1,3,2] => [1,3,3,3,2,2]
+     *
+     * Make sure the next PHPDoc instructions will be added:
+     * @param array $input
+     * @return array
+     */
+    public function repeatArrayValues(array $input): array
+    {
+        $array = [];
+        foreach($input as $value){
```

Comment:
Please refactor it according to the [PSR-12](https://www.php-fig.org/psr/psr-12/)